server/models/Tokens.spec.ts

```diff
@@ -0,0 +1,25 @@
+import TokensModel from './Tokens';
```

Comment:
```suggestion
import TokensModel from './tokens';
```

Eslint detected this error